data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java

```diff
@@ -144,10 +145,10 @@ private void registerControllers(ServiceExtensionContext serviceExtensionContext
         List<Handler> handlers = new LinkedList<>();
         handlers.add(descriptionHandler);
 
-        TransferProcessStore transferProcessStore = serviceExtensionContext.getService(TransferProcessStore.class);
+        TransferProcessManager transferProcessManager = serviceExtensionContext.getService(TransferProcessManager.class);
```

Comment:
Maybe you can update the required set in `IdsMultipartApiServiceExtension`, too. There is still the `TransferProcessStore` in it.

```java
@Override
    public Set<String> requires() {
        return Set.of(IdentityService.FEATURE,
                "dataspaceconnector:transferprocessstore",
                "edc:ids:core",
                "edc:ids:transform:v1");
    }
```